### PR TITLE
Add cloud-init foo for configuring Tenable Nessus Agent

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request adds some cloud-init foo to properly configure the Tenable Nessus Agent.

## 💭 Motivation and context ##

The Tenable Nessus Agent will not operate as expected if it is not properly configured.

## 🧪 Testing ##

All automated tests pass.  I used these changes to deploy the OpenVPN server to our staging COOL environment and verified that the server was configured as expected.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,6 +119,11 @@ module "example" {
 | freeipa\_domain | The domain for the IPA client (e.g. example.com). | `string` | n/a | yes |
 | freeipa\_realm | The realm for the IPA client (e.g. EXAMPLE.COM). | `string` | n/a | yes |
 | hostname | The hostname of the OpenVPN server (e.g. vpn.example.com). | `string` | n/a | yes |
+| nessus\_agent\_install\_path | The install path of Nessus Agent (e.g. /opt/nessus\_agent). | `string` | `"/opt/nessus_agent"` | no |
```
Comment (mcdonnnj) [inline README.md]:
```suggestion
| nessus\_agent\_install\_path | The install path of the Nessus Agent (e.g. /opt/nessus\_agent). | `string` | `"/opt/nessus_agent"` | no |
```

File: variables.tf [hunk 2]
```diff
@@ -132,6 +147,18 @@ variable "crowdstrike_falcon_sensor_install_path" {
   default     = "/opt/CrowdStrike"
 }
 
+variable "nessus_agent_install_path" {
+  type        = string
+  description = "The install path of Nessus Agent (e.g. /opt/nessus_agent)."
```
Comment (mcdonnnj) [inline variables.tf]:
```suggestion
  description = "The install path of the Nessus Agent (e.g. /opt/nessus_agent)."
```

File: cloudinit/link-nessus-agent.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    # Create STS client
+    sts_client = boto3.client("sts")
+
+    # Assume the role that can read the SSM Parameter Store parameters
+    stsresponse: dict[str, Any] = sts_client.assume_role(
+        RoleArn=SSM_READ_ROLE_ARN, RoleSessionName="nessus_agent_linking"
+    )
+    newsession_id = stsresponse["Credentials"]["AccessKeyId"]
+    newsession_key = stsresponse["Credentials"]["SecretAccessKey"]
+    newsession_token = stsresponse["Credentials"]["SessionToken"]
```
Comment (mcdonnnj) [inline cloudinit/link-nessus-agent.py]:
```suggestion
    newsession_id: str = stsresponse["Credentials"]["AccessKeyId"]
    newsession_key: str = stsresponse["Credentials"]["SecretAccessKey"]
    newsession_token: str = stsresponse["Credentials"]["SessionToken"]
```

**Review by mcdonnnj — APPROVED** [review]
LGTM ✔ Some minor suggestions for your consideration.

**Review by dav3r — APPROVED** [review]
👍 👍

**Pull request merged**